core/src/main/scala/sttp/tapir/EndpointIO.scala

```diff
@@ -218,6 +218,8 @@ object EndpointInput extends EndpointInputMacros {
 
     def description(d: String): Auth[T, TYPE] = copy(info = info.description(d))
 
+    def bearerFormat(f: String): Auth[T, TYPE] = copy(info = info.bearerFormat(f))
```

Comment:
one thing we might do is add a constraint here on `TYPE` to ensure that we're calling it on AuthType.Http only, using sth like `(implicit typeIsHttp: TYPE =:= AuthType.Http)`